packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -176,6 +176,97 @@ export async function uploadFile(node: ZoweDatasetNode, doc: vscode.TextDocument
     }
 }
 
+/**
+ * Deletes nodes from the data set tree & delegates deletion of data sets, members, and profiles
+ *
+ * @export
+ * @param {IZoweDatasetTreeNode} node - The node selected for deletion
+ * @param {DatasetTree} datasetProvider - the tree which contains the nodes
+ */
+export async function deleteDatasetPrompt(datasetProvider: IZoweTree<IZoweDatasetTreeNode>) {
+    const treeView = datasetProvider.getTreeView();
+    const selectedNodes: IZoweDatasetTreeNode[] = treeView.selection;
+
+    // Filter out sessions, favorite nodes, or information messages
+    const nodes: IZoweDatasetTreeNode[] = selectedNodes.filter(
+        (selectedNode) =>
+            selectedNode.getParent() &&
+            !contextually.isFavorite(selectedNode) &&
+            !contextually.isFavorite(selectedNode.getParent()) &&
+            !contextually.isSession(selectedNode) &&
+            !contextually.isInformation(selectedNode)
+    );
+
+    // The names of the nodes that should be deleted
+    const deletedNodes: string[] = nodes.map((deletedNode) => {
+        return contextually.isDsMember(deletedNode)
+            ? ` ${deletedNode.getParent().getLabel()}(${deletedNode.getLabel()})`
+            : ` ${deletedNode.getLabel()}`;
+    });
+
+    // Confirm that the user really wants to delete
+    globals.LOG.debug(localize("deleteDatasetPrompt.log.debug", "Deleting data set(s): ") + deletedNodes.join(","));
+    const quickPickOptions: vscode.QuickPickOptions = {
+        placeHolder: localize(
+            "deleteDatasetPrompt.quickPickOption",
+            "Delete data set(s)? This will permanently remove them from your system."
```

Comment:
This is just something small that I noticed: it was a bit weird seeing `delete data sets` when I am deleting members. I understand, though, that they share this same quickpick.
It is not a big deal for me since I am working under the data sets tree, and the success info message clearly specifies which members were deleted.